crates/guest/Cargo.toml

```diff
@@ -14,7 +14,7 @@ crate-type = ["cdylib", "rlib"]
 path = "src/guest.rs"
 
 [dependencies]
-holochain_serialized_bytes = { version = "=0.0.55", features = [] }
+holochain_serialized_bytes = "=0.0.55"
```

Comment:
Unrelated clean up. Cargo features are additive and this dep does not specify `no-default-features`, so I believe the `features = []` has no effect.